config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -168,7 +169,6 @@ presubmits:
         - /workspace/scenarios/kubernetes_e2e.py
         args:
         - --build=quick
-        - --extract=local
```

Comment:
we only dropped the extract, we should drop the `--stage` (this is the one that should fix the job)

Reply:
`and` o `or`

Reply:
And